examples/connected-redux/src/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
do you think it'd make sense to put everything here, for this example specifically? it's how i have it in the others... i think it'd still be short enough to be readable